guides/app-developers/bridge-tokens.md

```diff
@@ -1,6 +1,8 @@
+## Beta Testnet
 
+Visit [the bridge](https://bridge.beta.testnet.l2.quarkchain.io) and follow through the UI.
 
-## Testnet
+## Alpha Testnet
```

Comment:
The bridge for the alpha testnet has been discontinued. I suggest we avoid mentioning it.

Reply:
Removed in e1feddb176b117f2919c031631f7ee8d56afaa0e .